packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,44 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// {@tool dartpad --template=stateful_widget_material}
```

Comment:
Let's use a scaffold center template:
```suggestion
/// {@tool dartpad --template=stateful_widget_scaffold_center}
```